java-management/set-up-of-management-agent/set-up-of-management-agent.md

```diff
@@ -49,7 +49,7 @@ Select the correct compartment where your instance is located. If you have follo
 - Click on the **Managed instances** under Resources on the left hand side.
 - Click on the **Add managed instance(s)** button. A side panel of all the unregistered managed instances should show up.
 - Select the compartment where the unregistered managed instances is located. If you have followed the instruction in [Lab 4](?lab=create-a-java-application), this should be **Fleet_Compartment**.
-- Your managed instance from Step 3 should show up. Check the box next to your instance, then click **Add** to register this instance to your fleet.
+- Your managed instance from Step 3 should appear. Check the box next to your instance, or manually add an OCI instance, , then click **Add** to register this instance to your fleet.
```

Comment:
Instead of screenshot for "manually add an OCI instance", I prefer you show screenshot for 
"Your managed instance from Step 3 should appear. Check the box next to your instance"